src/Chart.jsx

```diff
@@ -62,6 +62,11 @@ class Chart extends Component {
 
   createChart() {
     const lineColor = '#71a0ae';
+    var foWidth = 300;
```

Comment:
We could use es6 syntax here. Ex, `const`. Also, run `yarn precommit` and it should do standard formatting of the javascript.